document/CoreFormat.mdk

```diff
@@ -839,70 +832,61 @@ This allows tools to include information about the themselves that is not explic
 
 ### General { #file-General }
 
-A `file` object represents a single file. In some cases, this file might be nested within another file,
-referred to as the "containing file."
-
-`file` objects occur as members of arrays within the `run.files` object (&sect;[#run-files]). 
-
-If the `file` object does not represent a nested file, neither of the properties
-`uri` (&sect;[#file-uri]) or `offset` (&sect;[#file-offset]) shall be present.
-
-If the location of a nested file can be expressed either by means of a path within the containing file
-or by means of a byte offset from the start of the containing file, then either `uri`
-or `offset` (&sect;[#file-offset]) or both shall be present; they shall not both be absent.
```

Comment:
I moved variants of this paragraph down into the sections on the `uri` and `offset` properties. It interacted to closely with the rest of the descriptions of those properties to be separated from them, even though the previous arrangement was slightly DRYer.